src/constants/axis_placeable_objects.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
This file is not really belong to our `constants`.
Let's move & rename this file to `src/components/annotations/axis_ref.js` and also place `getRefType` into that file please.

Reply:
Perhaps it shouldn't go in `src/constants` but it is also used by `src/components/images` and `src/components/shapes`, too. So maybe in `src/components` somewhere then?

Reply:
Once placed in `annotations` we would still be able to access it like this from `shapes` and `images`:
```js
var axisRef = require('../annotations/axis_ref');
```

Reply:
It's true but it's more of a question of respecting the already established organization where everything specific to `annotations` is in its folder etc. I mean why not put it in `src/components/shapes` instead?

Reply:
Good question. `shapes` already reuse parts of the code from `annotations`:
https://github.com/plotly/plotly.js/blob/86f5e2d02d56a94fcf2c26b30d3539fd54211c40/src/components/shapes/attributes.js#L11

Reply:
@nicholas-esterer are you going to address this?

Reply:
Well yes and no. I can take it out of `'src/constants'`, but I really don't understand what it has to do with `axis_ref` and `getRefType`. This latter function is really related to axes and not at all to documentation strings. The file we are talking about `axis_placeable_objects` generates a documentation string. Also the function has to do with all the `image`, `shape` and `annotation` objects. So I'd like maybe a second opinion on this one.